launch/esw.launch

```diff
@@ -0,0 +1,9 @@
+<launch>
+  <!--
+    Run the drive programs
+  -->
+  <node name="odrive_front" pkg="mrover" type="odrive_control.py" args="front"></node>
```

Comment:
to be really honest, I think a lot of the ODrive code could use a refactor as opposed to just being ported directly over to ROS. Since we are moving toward replacing our ODrives I won't be too much of a stickler about it right now. 

However, I would really like if we could make the entire set of drive motors be controlled by a single ROS node.  Launching 3 separate processes to control the motors of the drivetrain of our robot is quite silly.

Reply:
Yeah no I totally agree. The only issue is that if we used one script, we still need "multiple threads" because the "find odrive" command is a blocking function. Using "multiple threads" using the python libraries usually sucks and slows down performance a lot so we tried it in the past and didn't have that much success.

I think @Polishdudealan was messing around with rewriting the blocking function but at that point, going this far to rewrite the library made the code look very sketchy and overly complicated

Reply:
I mean I had some code that worked - tested on 2 odrives (only one had motors). Never got to testing on the full rover setup because it required the firmware to be 3.5 but we had reflashed the rover odrives to be on like 3.2 or something for reasons.

I agree that with the new code we should definitely only have one ROS node and one process to control the drive system.

Reply:
Can you elaborate on what's blocking? Do we need true multi-threading or will asyncio/coroutines work?

Reply:
So when we try to locate the odrive via USB connection, we use the "find odrive" function. This establishes a communication (so if u disconnect the usb, no communication will be established). This is the blocking function. As for the the second part of your question, I'm not sure, but the only info i can give you id that i think its not an async function

Reply:
If you want the actual line, it is line 512 in odrive_control.py and the function is "find_any(.........)"